proxy/http/HttpConfig.cc

```diff
@@ -179,6 +179,33 @@ http_server_session_sharing_cb(const char *name, RecDataT dtype, RecData data, v
   return REC_ERR_OKAY;
 }
 
+static int
+http_insert_forwarded_cb(const char *name, RecDataT dtype, RecData data, void *cookie)
+{
+  bool valid_p        = false;
+  HttpConfigParams *c = static_cast<HttpConfigParams *>(cookie);
+
+  if (0 == strcasecmp("proxy.config.http.insert_forwarded", name)) {
+    if (RECD_STRING == dtype) {
+      std::string error;
+      HttpForwarded::OptionBitSet bs = HttpForwarded::optStrToBitset(ts::string_view(data.rec_string), error);
+      if ("" == error) {
```

Comment:
error.empty() no? Actually, I think error should just be a const char* (see above).